test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -2227,6 +2227,12 @@ public void PackCommand_InstallPackageToOutputPath()
                     Assert.Equal("https://github.com/NuGet/NuGet.Client.git", nuspecReader.GetRepositoryMetadata().Url);
                     Assert.Equal("dev", nuspecReader.GetRepositoryMetadata().Branch);
                     Assert.Equal("e1c65e4524cd70ee6e22abe33e6cb6ec73938cb3", nuspecReader.GetRepositoryMetadata().Commit);
+                    // Validate the nuspec round trips.
+                    using (var nuspecStream = nupkgReader.GetStream($"packageA.nuspec"))
+                    {
+                        var manifest = Packaging.Manifest.ReadFrom(nuspecStream, true);
```

Comment:
`validateSchema: true`

Also assert that the return value is valid, even if only not null.